.github/workflows/integration-tests.yml

```diff
@@ -13,6 +13,9 @@ on:
         description: 'PHAR checksum from reproducible build (please rebase)'
         required: true
         type: string
+  push:
+    branches:
+      - "*"
```

Comment:
Not this

Reply:
Definitely, but how can I trigger/debug the test otherwise?

I need to debug/tweak the `open_basedir` path.

Reply:
You need to tweak this (it's hardcoded to take integration-tests.yml from 1.7.x here): https://github.com/phpstan/phpstan/blob/7c465c20f2b6ff0197c4232017036113099d44c3/.github/workflows/integration-tests-run.yml#L34